.github/workflows/cgl.yaml

```diff
@@ -26,15 +26,19 @@ jobs:
 
       # Validation
       - name: Validate composer.json
-        run: composer validate --no-check-lock
+        run: composer validate
 
       # Install dependencies
       - name: Install Composer dependencies
-        uses: ramsey/composer-install@v2
+        run: composer require --no-progress gmostafa/php-graphql-client:"^1.3"
```

Comment:
🤔

Reply:
This is required for the composer-require-checker to work properly. Have a look at CPS-IT/project-builder where we're doing the same thing.